reactor-netty-core/src/main/java/reactor/netty/tcp/SslProvider.java

```diff
@@ -243,7 +243,10 @@ public interface SslContextSpec {
 		 * @param spec SslContext builder that provides, specific for the protocol, default configuration
 		 * @return {@literal this}
 		 * @since 1.0.6
+		 * @deprecated as of 1.2.0. Prefer using {@link #sslContext(GenericSslContextSpec)}.
```

Comment:
when generating the javadoc, I noticed that the suggested link is not clickable, see:

![Screenshot 2024-04-16 at 12 02 54](https://github.com/reactor/reactor-netty/assets/650758/68ff11e5-0311-47c6-a7fd-3f7cc9ce6882)

The following fixes the link:

```suggestion
		 * @deprecated as of 1.2.0. Prefer using {@link #sslContext(SslProvider.GenericSslContextSpec)}.
```

Reply:
nice catch, fixed